### PR TITLE
66 database seeding example

### PR DESCRIPTION
**Pull request by dan-barrett**

### CODE REVIEW

File: examples/database-seeding/arc.env.dev.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+services:
+  examples/database-seeding:latest:
+    debug:
+      path: .
+    ingress:
+      subdomain: api
+    parameters:
+      AUTO_DDL: migrate
```
Comment (dan-barrett) [inline examples/database-seeding/arc.env.dev.yml:8]: @davidthor @tjhiggins 

Do we have open thoughts on environment config naming conventions when we have several of them next to each other like this?

For this demo I called them this:
```
arc.env.dev.yml
arc.env.qa.yml
arc.env.prod.yml
```

Would like to settle on something even if it's not this so we're consistent.

Comment (tjhiggins) [reply examples/database-seeding/arc.env.dev.yml:8]: It's intuitive enough. Fine for now and other examples.

File: examples/database-seeding/arc.env.prod.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    ingress:
+      subdomain: api
+    parameters:
+      AUTO_DDL: none
+    dependencies:
+      my-demo-db:
+        interfaces:
+          postgres:
+            host: host.docker.internal
+            port: 5432
```
Comment (davidthor) [inline examples/database-seeding/arc.env.prod.yml:14]: I'm not sure this is a particularly useful file. I know it shows host-overrides, but the file cruft may distract from the specifics of this particular example. I'd recommend we only have 1 env file with a comment showing how they can change the AUTO_DDL value.

File: examples/database-seeding/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "database-seeding",
+  "version": "1.0.0",
+  "description": "Example app to exhibit database seeding with architect",
+  "main": "index.js",
+  "scripts": {
+    "start": "node src/app.js",
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (davidthor) [inline examples/database-seeding/package.json:8]: Probably should remove this line.

File: examples/database-seeding/README.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+##### The 'prod' environment:
+```
+architect deploy -l arc.env.prod.yml
+```
+
+This environment is configured to connect to an existing database on `localhost:5432` (referenced by `host.docker.internal`) and has the AUTO_DDL property set to 'none'. NOTE: the `/users` endpoint will return an error unless you've preconfigured the schema in that external database.
+
+
+### Example Explained
```
Comment (davidthor) [inline examples/database-seeding/README.md:25]: I'd add one more section under the "explained" header that shows where the fixtures are and refers to the TypeORM docs.

**Comment by dan-barrett** [comment]
@davidthor - made the above adjustments, good to go pending tests

**Pull request merged**